src/main/java/com/acrolinx/proxy/AcrolinxProxyHttpServlet.java

```diff
@@ -214,9 +212,19 @@ private void configureTimeouts(HttpRequestBase httpRequestBase) {
             .build());
   }
 
-  private String getInitParameterOrDefaultValue(final String name, final String defaultValue) {
+  private String getInitParameterOrDefaultValue(String name, String defaultValue) {
     String parameterValue = getInitParameter(name);
-    return parameterValue != null ? parameterValue : defaultValue;
+    return parameterValue == null ? defaultValue : parameterValue;
+  }
+
+  private String getInitParameterOrDefaultValue(final String name) {
```

Comment:
the method name `getInitParameterOrDefaultValue()` is a bit misleading if the _default value_ means a thrown exception ;-)

I would suggest `getInitParameterOrThrowException` or something like that